src/services/assets/assets.svc.coffee

```diff
@@ -15,6 +15,12 @@ angular
       impacTitleLogo: ':default/impac-title-logo.png'
       impacDashboardBackground: ':default/impac-dashboard-background.png'
       currencyConversionsIcon: ':default/currency-conversions.png'
+      cashFlowLegendIcon: ':default/cash-flow.png'
+      payablesLegendIcon: ':default/payables.png'
+      projectedCashLegendIcon: ':default/projected-cash.png'
+      receivablesLegendIcon: ':default/receivables.png'
+      plotLineLegendIcon: ':default/plot-line-icon.svg'
+      areaLegendIcon: ':default/area-icon.svg'
```

Comment:
@cesar-tonnoir I think we were going to wait for Tristan's review. I can make them svg today so that the decision can be made. There is also the question of whether to use a custom legend for the cash-projection widget like the cash-balance widget.